types/smokehouse.d.ts

```diff
@@ -19,17 +19,25 @@
     diff?: Difference | null;
   }
 
-  export type ExpectedLHR = Pick<LH.Result, 'audits' | 'finalUrl' | 'requestedUrl' | 'runtimeError'>
+  interface ExpectedLHR {
+    audits: Record<string, any>;
+    requestedUrl: string;
+    finalUrl: string;
+    runtimeError?: {
+      code?: any;
+      message?: any;
+    };
+  }
 
   export type ExpectedRunnerResult = {
     lhr: ExpectedLHR,
-    artifacts?: Partial<LH.Artifacts>
+    artifacts?: Partial<Record<keyof LH.Artifacts, any>>
   }
 
-  export interface TestDfn {
+  export interface Test {
```

Comment:
I think I prefer `TestDefn`. It's the definition of a test that something else will run, while a `Test` sounds like something that will be doing the testing, at least to me :)

Reply:
Sounds convincing.